server/src/modules/jobs/jobs.ts

```diff
@@ -74,6 +75,13 @@ export async function setupJobProcessor() {
               resumable: true,
               maxRuntimeInMinutes: 12 * 60,
             },
+            "Vérfication des emails des rupturants du TDB": {
+              cron_string: "0 2 * * 6",
+              handler: async () => {
+                await addJob({ name: "job:tdb:insert-process-pending-email", queued: true });
+                return 0;
```

Comment:
Mets les fonctions directement dans le handler, ça permet d'avoir une erreur sur le cron status directement dans Sentry.
```suggestion
               await hydrateEffectifsEmail();
               await processEffectifsPendingMail();
                return 0;
```

Reply:
Ok, je laisse quand même le job si on doit l’appeler manuellement ( ce qui va arriver au moins en recette pour faire des tests d'intégration)

Reply:
Yes 👍